src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewUnrealizedChildrenTestPage.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using MUXControlsTestApp.Utilities;
+
+namespace UITests.Shared.Microsoft_UI_Xaml_Controls.TreeViewTests
+{
+	public sealed partial class TreeViewUnrealizedChildrenTestPage : MUXTestPage
+	{
+
+		public TreeViewNode VirtualizingTestRootNode;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)